src/ComServerHelpers/BaseActivationFactory.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.Versioning;
+
+namespace ComServerHelpers;
+
+/// <summary>
+/// Base for a WinRT Activation Factory for a .NET type.
+/// </summary>
+[SupportedOSPlatform("windows8.0")]
+public abstract class BaseActivationFactory
```

Comment:
Can these be internal or no?

Reply:
Don't think so -> all the factory classes are used in public extension methods like https://github.com/shmuelie/Shmuelie.WinRTServer/blob/12fe717ff4ce3e74e677f0bf4578f73f15ba24df/src/Shmuelie.WinRTServer/WinRtServerExtensions.cs#L61
@shmuelie to confirm

Reply:
They're public so that you're not forced to use the factories we provide. Using it would be a more advanced scenario